### PR TITLE
fix(yaml) K8S YAML should contain mesh and namespace

### PR DESCRIPTION
**Pull request by jakubdyszkiewicz**
Two things
* K8S YAML should contain a mesh (if there is a Mesh, some objects are global scoped)
* K8S YAML should contain a namespace if the name is `name.namespace` from Kuma API

### CODE REVIEW

File: src/components/Skeletons/YamlView.vue [hunk 1]
```diff
@@ -224,15 +224,24 @@ export default {
...
         // assemble the main part of our object
         newObj.apiVersion = 'kuma.io/v1alpha1'
         newObj.kind = type
-        newObj.metadata = {
-          name: name
+        if (mesh !== undefined) { // mesh can be empty on global scoped objects
+          newObj.mesh = sourceObj.mesh
+        }
+
+        if (name.includes('.')) { // if name from Kuma has '.' it means it's k8s name joined with a namespace by dot
```
Comment (nickolaev) [inline src/components/Skeletons/YamlView.vue:234]: OK will this be working if we introduce the "parametrized' naming of services? Not sure if this aplies though

File: src/components/Skeletons/YamlView.vue [hunk 2]
```diff
@@ -224,15 +224,24 @@ export default {
...
-        // we remove the `name` and `type` because they need to be placed under `metadata`
-        delete sourceObj.type
-        delete sourceObj.name
-
         // assemble the main part of our object
         newObj.apiVersion = 'kuma.io/v1alpha1'
         newObj.kind = type
-        newObj.metadata = {
-          name: name
+        if (mesh !== undefined) { // mesh can be empty on global scoped objects
```
Comment (lobkovilya) [inline src/components/Skeletons/YamlView.vue:230]: If `mesh` is empty, why do we need this field in `newObj`?

File: src/components/Skeletons/YamlView.vue [hunk 3]
```diff
@@ -224,15 +224,24 @@ export default {
...
+          newObj.mesh = sourceObj.mesh
+        }
+
+        if (name.includes('.')) { // if name from Kuma has '.' it means it's k8s name joined with a namespace by dot
+          const [k8sName, namespace] = name.split('.')
```
Comment (lobkovilya) [inline src/components/Skeletons/YamlView.vue:235]: is it okay for multicluster where we sometimes have 2 dots in the name: `<name>.<original_ns>.default`?

File: src/components/Skeletons/YamlView.vue (hunk 1 above)
Comment (jakubdyszkiewicz) [reply src/components/Skeletons/YamlView.vue:234]: it has nothing to do with parametrized naming of services. It's about naming our resources

**Review by lobkovilya — APPROVED** [review]

**Pull request merged**